azext_edge/edge/providers/support/base.py

```diff
@@ -603,6 +663,16 @@ def _process_kubernetes_resources(
     return processed
 
 
+def _get_resource_type_prefix(kind: str) -> str:
+    if len(kind) > 12:
```

Comment:
What are the kinds that are true for this condition? Cluster role binding and what else?

Reply:
also persistentvolumnclaim